tests/lighthouse/run

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+#Old python script
+#python3 tests/lighthouse/evaluate.py
+
+#Old lighthouse command
+#lighthouse $TEST_ENV --output=json --output-path=./tests/lighthouse/reports/report.json --save-assets --chrome-flags="--headless"
+
+TEST_ENV=$(./tests/geturl $1)
+
+#Begin
+echo "Begin Lighthouse setup and healthcheck..."
+
+# set up key needed for google-chrome-stable
+wget -q -O - https://dl-ssl.google.com/linux/linux_signing_key.pub | apt-key add -
+
+# set up repository for google-chrome-stable
+sh -c 'echo "deb [arch=amd64] http://dl.google.com/linux/chrome/deb/ stable main" >> /etc/apt/sources.list.d/google.list'
+
+# install script dependencies
+apt-get update
+apt-get install google-chrome-stable -y
+
+# validate installation
+lhci healthcheck
```

Comment:
Is all of this really needed? It seemed to work fine without it before. I think this is only required if your docker image doesn't have the dependencies already installed, but I think ours does.

Reply:
I wasn't  sure about the docker image, I suggest leaving only the `lhci healthcheck` only to log any issue on the environment setup.